benchmarks/smallmatvec/manual_versions/libxsmm/results/libxsmm_results_h32_c32_schstatic_affscatter_smt1_r1.txt

```diff
@@ -0,0 +1,12 @@
+32 10 static scatter 32 1  0.379426956176758 1.231866950161898E+019  0.394292116165161 1.231866950161898E+019  0.358317136764526 1.236316606965435E+019
```

Comment:
We need to properly document/comment this file:
Hardware details, compiler details, date, what all the numbers mean!

Reply:
I've done this in a README file